content/docs/introduction/faq.md

```diff
@@ -40,14 +43,16 @@ Python, and Ruby.
 
 ### How stable are Prometheus features, storage formats, and APIs?
 
-Although Prometheus and many of its ecosystem components are already quite
-stable, we will still allow for occasional breaking changes until the
-Prometheus server reaches version 1.0.0. These breaking changes will be pointed
-out in release announcements for components that already have a proper release
-process (like the Prometheus server) or communicated clearly otherwise. After
-releasing version 1.0.0, breaking changes will be indicated by increments of
-the major version. See also the documentation for [semantic
-versioning](http://semver.org/), which we are following.
+All components of the ecosystem that have reached version 1.0.0 follow
+[semantic versioning](http://semver.org/). Breaking changes are indicated by
+increments of the major version. Exceptions are possible for experimental
+components, which are clearly marked as such in announcements.
+
+Even components that have not yet reached version 1.0.0 are in general quite
```

Comment:
The 1.0.0 reference here is confusing, I'd make it clearer that this is about repos rather than components of Prometheus itself.